일정에 등록한 수행자를 삭제하려고 하면 500 에러가 발생한다

### 🐞 설명

일정에 등록한 수행자를 삭제하려고 하면 500 에러가 발생하고 있습니다.

![Image](https://github.com/user-attachments/assets/31d56484-eeb5-4e41-8e2c-6e5befb1d631)

### 📄 버그 로그

```shell
[ERROR] (DELETE /api/v1/project/1/task/5/assignee/2) could not execute statement [(conn=154940) Unknown column 'id' in 'where clause'] [UPDATE task_users SET is_deleted = true WHERE id = ?]; SQL [UPDATE task_users SET is_deleted = true WHERE id = ?]
```

### 🌏 버그 발생 환경

Ubuntu 24.04, Firefox 134

### 🙋🏻 버그 재현 방법

Given: 일정에 수행자를 등록합니다.
When: 일정을 누르고 수행자를 제거하려고 시도합니다.
Then: 사용자에게 "수행자 삭제에 실패했습니다. 잠시 후 다시 시도해주세요." 메시지가 보여지고 500 에러가 전송됩니다.

### ⏰ 추정 시간

30분

### ⏰ 소요 시간

5시간

--- a/src/main/java/com/growup/pms/task/domain/TaskUser.java
+++ b/src/main/java/com/growup/pms/task/domain/TaskUser.java
@@ -21,7 +21,7 @@ import org.hibernate.annotations.SQLRestriction;
 @Getter
 @IdClass(TaskUserId.class)
 @Table(name = "task_users")
-@SQLDelete(sql = "UPDATE task_users SET is_deleted = true WHERE id = ?")
+@SQLDelete(sql = "UPDATE task_users SET is_deleted = true WHERE user_id = ? AND task_id = ?")
 @SQLRestriction("is_deleted = false")
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class TaskUser extends BaseEntity {

